proto/nameserver2.proto

```diff
@@ -425,6 +425,13 @@ message OpenFileRequest {
     optional string signature = 3;
     required uint64 date = 4;
     optional string clientVersion = 5;
+    // make the MDS know isfakewriter
+    optional bool isFakeWriter = 6;
+    // only the client want to be writer
+    // the mds will try Lock
+    optional bool tobewriter = 7;
+    optional string clientIP = 8;
+    optional uint32 clientPort = 9;
```

Comment:
it's not a good way to identify an owner by IP and Port, a feasible approach might be using a UUID. 

For example, when a client opens a file, get a UUID for it. MDS saves UUID which opens with write permission, and client also uses the UUID in refresh session requests.

Reply:
Good idea! i have fixed it, and now when client open file, he will get the uuid (48bytes). the etcd save the key : filename, value : clientuuid + writerlastheartbeat.